packages/cli/src/utils/get-latest-version.ts

```diff
@@ -1,24 +1,36 @@
 import {fetchAllVersions, fetchLatestDistVersion} from './npm-fetch'
+import {log} from '@blitzjs/server'
 
-export const getLatestVersion = async (dependency: string, templateVersion: string) => {
+/**
+ * @example const [ version, isFallback ] = await getLatestVersion("blitz", "2.x");
```

Comment:
I like the new explicit API! Can we update these utilities to return an object that conforms to something like an `interface NpmResult {dependency: string; falback: boolean, version: string}` so the return value is a little more explicit? Right now you have to read into the function to really understand what the return value signifies

Reply:
I agree with you that an object is clearly a better choice for readability, though I don't really understand why `dependency` should be part of it. Callers already know the dependency since they pass it into it in the first place, why should it be returned?

Reply:
Please have a look at 90722873e8439cec4b97583cb69a48f4614acc50, I think that captures it nicely.